src/ui/simulator/windows/options/optimization/optimization.cpp

```diff
@@ -708,23 +716,39 @@ void Optimization::onPopupMenuTransmissionCapacities(Component::Button&, wxMenu&
                           wxEmptyString);
     menu.Connect(it->GetId(),
                  wxEVT_COMMAND_MENU_SELECTED,
-                 wxCommandEventHandler(Optimization::onSelectTransCapInclude),
+                 wxCommandEventHandler(Optimization::onSelectTransmissionCapacity<Data::tncEnabled>),
```

Comment:
The body of **onPopupMenuTransmissionCapacities** could be 4 lines, that is 4 calls to a template function that creates item and put it in a menu, like it was recently done for MPS writers (see PR #872)

Reply:
Yes, we can go one step forward & simplify even more